### PR TITLE
RLS: switch master from 0.18.2 to 0.19.0

### PR DESCRIPTION
**Pull request by jorisvandenbossche**
xref #13562

### CODE REVIEW

File: doc/source/whatsnew/v0.19.0.txt [hunk 1]
```diff
@@ -1,83 +1,532 @@
 .. _whatsnew_0190:
 
-v0.19.0 (????, 2016)
---------------------
+v0.19.0 (August ??, 2016)
+-----------------------
```
Comment (jreback) [inline doc/source/whatsnew/v0.19.0.txt]: line len

**Comment by jreback** [comment]
prob pandas/doc/release.rst as well.

**Comment by jorisvandenbossche** [comment]
`release.rst` did not yet include anything about 0.18.2 (we always only add right before the release I think)

**Comment by jorisvandenbossche** [comment]
Merging, so other PRs can be rebased on to this with the new whatsnew file

**Pull request merged**